### PR TITLE
Liqun/modify verification config

### PR DESCRIPTION
**Pull request by liqul**
lift all verification configurations to code interpreter. now, code generator will receive these configurations through init process.

### CODE REVIEW

File: taskweaver/code_interpreter/code_generator/code_generator.py [hunk 1]
```diff
@@ -57,13 +56,17 @@ def __init__(
         plugin_registry: PluginRegistry,
         logger: TelemetryLogger,
         llm_api: LLMApi,
-        code_verification_config: CodeVerificationConfig,
         round_compressor: RoundCompressor,
+        plugin_only: bool = False,
+        code_verification_on: bool = False,
```
Comment (Jack-Q) [inline taskweaver/code_interpreter/code_generator/code_generator.py:61]: wondering why we need to have these configurations in constructor when we also have a `configure_verification` method

File: taskweaver/code_interpreter/code_generator/code_generator.py [hunk 2]
```diff
@@ -94,27 +100,39 @@ def __init__(
...
         self,
         plugin_list: List[PluginEntry],
     ) -> str:
         requirements = []
-        if not self.code_verification_config.code_verification_on:
+        if not self.code_verification_on:
             return ""
-        if self.code_verification_config.plugin_only:
+
+        if self.plugin_only:
```
[the remark this answers is not in the document]
Comment (liqul) [reply taskweaver/code_interpreter/code_generator/code_generator.py:121]: If plugin-only is turned on, verification will be turned on as well to ensure its correctness.

File: taskweaver/code_interpreter/code_generator/code_generator.py (hunk 1 above)
Comment (liqul) [reply taskweaver/code_interpreter/code_generator/code_generator.py]: removed

File: taskweaver/code_interpreter/code_generator/code_generator.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (liqul) [reply taskweaver/code_interpreter/code_generator/code_generator.py:115]: I agree. I still feel the configuration design is not perfect. But for now, let's keep it as it is and wait for future understanding to improve it.

**Review by Jack-Q — APPROVED** [review]

**Pull request merged**